src/emails/pix/conseillerRupturePix.ts

```diff
@@ -0,0 +1,34 @@
+import logger from '../../logger';
+
+export default function (mailer) {
+  const templateName = 'conseillerRupturePix';
+  const { utils } = mailer;
+
+  const render = async (conseiller) => {
+    return mailer.render(__dirname, templateName, { conseiller });
+  };
+
+  return {
+    templateName,
+    render,
+    send: async (conseiller) => {
+      const onSuccess = async () => {
+        logger.info(
+          `Email envoyé à PIX avec succès pour la suppression du conseiller ${conseiller.nom} ${conseiller.prenom}`,
+        );
+      };
+      const onError = async (err: Error) => {
+        throw err;
+      };
+
+      return mailer
+        .createMailer()
+        .sendEmail(utils.getPixContactMail(), {
```

Comment:
il me semble qu'il fallait le contact principal dans le to mais le support pix en Copie - j'avais du ajusté une config dans le mailer surement pour avoir le cc